wolfcrypt/src/coding.c

```diff
@@ -93,6 +93,11 @@ int Base64_Decode(const byte* in, word32 inLen, byte* out, word32* outLen)
             return ASN_INPUT_E;
         }
 
+        if (i + 3 > *outLen) {
```

Comment:
Don't we only write one byte to out[] sometimes depending on pad3 and pad4?

Reply:
Correct!
Fixed this to include pad3 and pad4 instead of just 3.